### PR TITLE
Change starting show screen progress bar condition

### PR DESCRIPTION
**Pull request by LeviHarrison**
Currently, we determine if the WAL replay has started by checking if the index of the current segment is greater than the index of the first segment (`min`). But, if we have just read the first segment, the values will be the same and the progress bar will not be shown even though it should. This PR changes the condition so we show the progress bar when the index of the final segment is not `0`, which means that a WAL replay has commenced (`max` is set when the replay starts and the default value is `0`). I realize that this still won’t work for the case when only the segment `0` needs to be loaded, but that requires some more involved changes.

### CODE REVIEW

**Comment by LeviHarrison** [comment]
@juliusv final change. I wrote this on an iPad so some things might be wrong 🤣

File: web/ui/react-app/src/components/withStartingIndicator.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const StartingContent: FC<StartingContentProps> = ({ status, isUnexpected
     <div className="text-center m-3">
       <div className="m-4">
         <h2>Starting up...</h2>
-        {status?.current! > status?.min! ? (
+        {status?.max! > 0 ? (
```
Comment (LeviHarrison) [inline web/ui/react-app/src/components/withStartingIndicator.tsx:26]: I know I’ve worded this change as “the max value is not 0”, but I think I’d be good to check if it is greater than 0, which yields the same affect unless something weird happens with the server.

**Comment by roidelapluie** [comment]
if there is only one segment hopefully it is not needed to have the progressbar.

**Comment by LeviHarrison** [comment]
I suppose so, just an edge case.

**Comment by roidelapluie** [comment]
Tests fail:

/home/circleci/project/web/ui/react-app/src/components/withStartingIndicator.test.tsx
  29:7  error  Insert `;`  prettier/prettier

**Comment by LeviHarrison** [comment]
Whoops, symptoms of iPad

**Comment by juliusv** [comment]
:+1: Thanks!

**Pull request merged**